es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+  },
+  {
+    "name": "FractionallySizedBox",
+    "description": "Un widget que dimensiona su hijo a una fracción del espacio total disponible. Para obtener más detalles sobre el algoritmo de diseño, consulte RenderFractionallySizedOverflowBox. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/FractionallySizedBox-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "IntrinsicHeight",
+    "description": "Un widget que dimensiona su hijo a la altura intrínseca del hijo. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/IntrinsicHeight-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "IntrinsicWidth",
+    "description": "Un widget que dimensiona su hijo al ancho intrínseco del hijo. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/IntrinsicWidth-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "LimitedBox",
+    "description": "Una caja que limita su tamaño solo cuando no está restringida. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/LimitedBox-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Offstage",
+    "description": "Un widget que establece al hijo como si estuviera en el árbol, pero sin pintar nada, sin que el hijo esté disponible para la prueba de golpes, y sin dejar espacio en el padre. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Offstage-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "OverflowBox",
+    "description": "Un widget que impone diferentes restricciones en su hijo de las que obtiene de su padre, permitiendo posiblemente que el hijo desborde al padre. ",
+    "categories": [
+    ],
+    "subcategories": [
+      "Single-child layout widgets"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/OverflowBox-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "SizedBox",
+    "description": "Una caja con un tamaño especificado. Si se le da un hijo, este widget fuerza a su hijo a tener un ancho y / o altura específicos (suponiendo que el padre de este widget permita los valores). Si el ancho o el alto es nulo, este artilugio se dimensionará a sí mismo para que coincida con el tamaño del niño en esa dimensión. ",
```

Comment:
A box with a specified size. If given a child, this widget forces its child to have a specific width and/or height (assuming values are permitted by this widget's parent). If either the width or height is null, this widget will size itself to match the child's size in that dimension.

Una caja con un tamaño específico. Si se le proporciona un hijo, este widget forzará a su hijo a tener una anchura y/o altura específicos ( asumiendo que los valores pueden ser permitidos por los padres de este widget). Si la anchura o la altura fueran null, este widget se dimensionará a si mismo para ajustarse al tamaño de sus hijos en esa dirección.